### ISSUE TITLE
References to other documents

### ISSUE THREAD

**Issue by iherman** [w3c/json-ld-bp#21]
Comment on:

https://github.com/w3c/json-ld-bp/blob/c79199120d74021cd7ba4bfc9f03c4c6434aab5a/index.html#L66

We have to be careful: these give the impression that the core constituency for JSON-LD is the linked data crowd. I am not sure this is what we want.

**Comment by ajs6f** [comment]
@iherman Would you agree that it is useful to refer to those other works, but that this is a matter of emphasis and placement? Perhaps we could make this information less prominent and explain that _JSON-LD is JSON_ first and foremost.

**Comment by iherman** [comment]
@ajs6f exactly. We should emphasize the JSON side, but also refer to those other works. The emphasis is what counts.

**Comment by ajs6f** [comment]
Ok, I think I got you. I'll try for a PR for this on Thursday. (Currently
out sick)

**Comment by ajs6f** [comment]
@iherman We're good on this after https://github.com/w3c/json-ld-bp/commit/bdf4fc0b607efd7ea36461c3c01b42d47ad969cb, right? If so, I'll close (or you can)!
